Makefile

```diff
@@ -83,7 +83,7 @@ test-others:
 	build/env.sh go run build/ci.go test -p 1 -exclude datasync,networks,tests
 
 cover:
-	build/env.sh go run build/ci.go test -coverage
+	build/env.sh go run build/ci.go cover -coverprofile=coverage.out
```

Comment:
As you created an independent function `cover` for coverage test, how about cleaning-up coverage-related codes from `doTest`?

Reply:
I cleaned up. Thank you!